modules/persistent-storage-csi-vsphere-disable-storage-overview.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-disable-storage-overview_{context}"]
+= Disabling and enabling storage on vSphere
+
+Cluster administrators might want to disable the VMWare vSphere Container Storage Interface (CSI) Driver Operator as a Day 2 operation, so the vSphere CSI Driver Operator does not interface with your vSphere setup.
```

Comment:
Let's remove the word "Operator" from both places:
```
Cluster administrators might want to disable the VMWare vSphere Container Storage Interface (CSI) Driver as a Day 2 operation, so the vSphere CSI Driver does not interface with your vSphere setup.
```
(operator is always enabled and ready to resurrect csi driver if state is set to "Managed" back)

Reply:
Done.